metrics/prometheus.go

```diff
@@ -2,12 +2,16 @@ package metrics
 
 import (
 	"path/filepath"
+	"regexp"
 	"strconv"
 	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/utilitywarehouse/kube-applier/log"
 )
 
+var r = regexp.MustCompile(`([\w.\-]+)\/([\w.\-]+) (\w+).*`)
```

Comment:
I think this variable should have a more descriptive name, something like `kubectlOutputPattern` maybe.